wd-monitoring-and-alerting.tf

```diff
@@ -6,7 +6,7 @@
 
 # Topic
 resource "aws_sns_topic" "waggle_dance_user_updates" {
-  name         = "waggle-dance-alerts"
+  name         = "${local.instance_alias}-alerts"
```

Comment:
What is a typical value for `local.instance_alias`? I'm just wondering if it would make sense for this to be `${local.instance_alias}-waggle-dance-alerts` so it's clear what type of alert this is (waggle-dance) as well as where it's coming from (local.instance_alias)?

Reply:
default value for local.instance_alias is "waggledance", other values will waggledance-<instancename> ..etc
so adding waggle-dance will make it redundant.

Reply:
OK, thanks for the clarification.